htdocs/multicurrency/multicurrency_rate.php

```diff
@@ -486,52 +448,69 @@
 
 		print '<tr class="oddeven">';
 
-		// date_sync
-		if (!empty($arrayfields['cr.date_sync']['checked']))
-		{
-			print '<td class="tdoverflowmax200">';
-			print $obj->date_sync;
-			print "</td>\n";
-			if (!$i) $totalarray['nbfield']++;
-		}
-
-		// code
-		if (!empty($arrayfields['m.code']['checked']))
-		{
-			print '<td class="tdoverflowmax200">';
-			print $obj->code." ".$obj->name;
-			print "</td>\n";
-
-			if (!$i) $totalarray['nbfield']++;
+        // USER REQUEST UPDATE FOR THIS LINE
+        if ($action == "updateRate" && $obj->rowid == $id_rate_selected) {
+            //  var_dump($obj);
+            print ' <td><input class="minwidth200" name="dateinput" value="'. date('Y-m-d', dol_stringtotime($obj->date_sync)) .'" type="date"></td>';
```

Comment:
type="date" must not be used since it is not compatible with the setup of user language language (the browser setup takes precendence).
Use instead the date selector of Dolibarr.

Reply:
input with type="date" must be replace with form->selectDate

Reply:
replaced to form->selectDate